src/experimental/Toast/Toast.features.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {Meta} from '@storybook/react'
+import BaseStyles from '../../BaseStyles'
+import ThemeProvider from '../../ThemeProvider'
+import {Box, Text} from '../..'
+import {ToastUI} from './ToastUI'
+
+const meta: Meta = {
+  title: 'Components/Toast/Features',
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Box sx={{display: 'inline-flex', flexDirection: 'column', gap: 3}}>{Story()}</Box>
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    },
+  ],
+  component: ToastUI,
+}
+export default meta
+
+export const Default = () => (
+  <>
+    <ToastUI variant="default">Message</ToastUI>
```

Comment:
This will swap out to the `<Toast />` with a trigger button once that's built.

Not sure if I should include a story for the internal ui component 🤔 if so, maybe this becomes `ToastUI.stories.tsx`?